docs/data-sources/gaussdb_nosql_flavors.md

```diff
@@ -0,0 +1,57 @@
+---
+subcategory: "GaussDB"
+---
+
+# huaweicloud_gaussdb_nosql_flavors
+
+Use this data source to get available HuaweiCloud GaussDB (for NoSQL) flavors.
+
+## Example Usage
+
+```hcl
+data "huaweicloud_gaussdb_nosql_flavors" "flavors" {
+  vcpus  = 4
+  memory = 8
+}
+```
+
+## Argument Reference
+
+* `region` - (Optional, String) Specifies the region in which to obtain the GaussDB specifications.
+  If omitted, the provider-level region will be used.
+
+* `engine_type` - (Optional, String) Specifies the type of the database engine. The valid values are as follows:
+  + **cassandra**: The default value and means to query GaussDB (for Cassandra) instance specifications.
+  + **mongodb**: Means to query GaussDB (for Mongo) instance specifications.
+  + **influxdb**: Means to query GaussDB (for Influx) instance specifications.
```

Comment:
suggest to put mongodb and influxdb after redis, as they does not have corresponding resources.

Reply:
sure